x-pack/plugins/observability_solution/observability/public/pages/alert_details/alert_details_contextual_insights.tsx

```diff
@@ -29,77 +28,25 @@ export function AlertDetailContextualInsights({ alert }: { alert: AlertData | nu
     }
 
     try {
-      const res = await http.get('/internal/apm/assistant/get_obs_alert_details_context', {
-        query: {
-          alert_started_at: new Date(alert.formatted.start).toISOString(),
-
-          // service fields
-          'service.name': fields['service.name'],
-          'service.environment': fields['service.environment'],
-          'transaction.type': fields['transaction.type'],
-          'transaction.name': fields['transaction.name'],
-
-          // infra fields
-          'host.name': fields['host.name'],
-          'container.id': fields['container.id'],
-          'kubernetes.pod.name': fields['kubernetes.pod.name'],
-        },
-      });
-
-      const {
-        serviceSummary,
-        downstreamDependencies,
-        logCategories,
-        serviceChangePoints,
-        exitSpanChangePoints,
-        anomalies,
-      } = res as any;
-
-      const serviceName = fields['service.name'];
-      const serviceEnvironment = fields['service.environment'];
-
-      const obsAlertContext = `${
-        !isEmpty(serviceSummary)
-          ? `Metadata for the service where the alert occurred:
-${JSON.stringify(serviceSummary, null, 2)}`
-          : ''
-      }
-
-    ${
-      !isEmpty(downstreamDependencies)
-        ? `Downstream dependencies from the service "${serviceName}". Problems in these services can negatively affect the performance of "${serviceName}":
-${JSON.stringify(downstreamDependencies, null, 2)}`
-        : ''
-    }
-    
-    ${
-      !isEmpty(serviceChangePoints)
-        ? `Significant change points for "${serviceName}". Use this to spot dips and spikes in throughput, latency and failure rate:
-    ${JSON.stringify(serviceChangePoints, null, 2)}`
-        : ''
-    }
-  
-    ${
-      !isEmpty(exitSpanChangePoints)
-        ? `Significant change points for the dependencies of "${serviceName}". Use this to spot dips or spikes in throughput, latency and failure rate for downstream dependencies:
-    ${JSON.stringify(exitSpanChangePoints, null, 2)}`
-        : ''
-    }
-    
-    ${
-      !isEmpty(logCategories)
-        ? `Log events occurring around the time of the alert:
-    ${JSON.stringify(logCategories, null, 2)}`
-        : ''
-    }
-  
-    ${
-      !isEmpty(anomalies)
-        ? `Anomalies for services running in the environment "${serviceEnvironment}":
-    ${anomalies}`
-        : ''
-    }          
-    `;
+      const { context: obsAlertContext } = await http.get<{ context: string }>(
+        '/internal/apm/assistant/get_obs_alert_details_context',
```

Comment:
I didn't make the change yet because I had some issues I still need to investigate when moving the api tests. Are we fine leaving the endpoint in apm and moving the endpoint as a follow up ?